packages/terra-form-validation/examples/index-examples/FormSubmitCheckboxField.jsx

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable class-methods-use-this */
+
+import React from 'react';
+import { Form, Field } from 'react-final-form';
+import Checkbox from 'terra-form-checkbox';
+import CheckboxField from 'terra-form-checkbox/lib/CheckboxField';
+import Button from 'terra-button';
+import Spacer from 'terra-spacer';
+
+const required = value => (value && value.length > 0 ? undefined : 'Required');
+
+export default class MainEntry extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {};
+    this.submitForm = this.submitForm.bind(this);
+  }
+
+  submitForm(values) {
+    this.setState({
+      submittedValues: values,
+    });
+  }
+
+  renderForm({ handleSubmit, errors }) {
```

Comment:
Let's switch validation on this to be onSubmit rather than onInput for this default example, I believe onSubmit is the more common use case UX intends. cc/ @neilpfeiffer